docs/how-to/integrate-external-libraries.md

```diff
@@ -0,0 +1,312 @@
+
+#### Step 1: Use FetchContent to include the library in your project
+
+The following excerpt demonstrates how to integrate the [ETL libraries](https://github.com/ETLCPP/etl) using `FetchContent`:
+
+```cmake
+# This code can be located in the root project.cmake file if the library is used by multiple modules
+# or directly in the module's CMakeLists.txt if it is only used by that module.
+include(FetchContent)
+FetchContent_Declare(
+    etl
+    GIT_REPOSITORY https://github.com/ETLCPP/etl
+    GIT_TAG        20.40.0
+)
+FetchContent_MakeAvailable(etl)
+```
+
+The same considerations as [Approach 1](#add_subdirectory) apply with regards to where to place that code.
+
+#### Step 2: Set the library as a dependency of the module
+
+Any component or module that depends on ETL can now link against it by adding the following entry to its `MOD_DEPS`:
+
+```cmake
+# In MyComponent/CMakeLists.txt 
+set(SOURCE_FILES
+  "${CMAKE_CURRENT_LIST_DIR}/MyComponent.fpp"
+  "${CMAKE_CURRENT_LIST_DIR}/MyComponent.cpp"
+)
+set(MOD_DEPS
+  etl::etl   # This is the target name for the ETL library
+)
+register_fprime_module()
+```
+
+
+---
+
+<a id="mod_deps"></a>
+
+## Approach 2: MOD_DEPS
+
+|     |     |
+| --- | --- |
+| **Benefits** | Easy to set up; no additional build time |
+| **Drawbacks** | Not portable across platforms |
+| **Considerations** | May be provided by a vendor or third-party |
+
+A pre-compiled library file is a library that has already been compiled and is ready to be used, often named `lib<libName>.a` or `lib<libName>.so`. There are many ways to obtain pre-compiled libraries, such as downloading them from a vendor repository or building them from source yourself.
+
+To integrate a pre-compiled library, you need to add the path of that library file to `MOD_DEPS` of the module(s) that depend on it (MOD_DEPS: module dependencies). The following example demonstrates how to integrate the OpenSSL `libcrypto` library into an F´ wrapper component:
+
+```cmake
+set(SOURCE_FILES
+    "${CMAKE_CURRENT_LIST_DIR}/OpenSslWrapper.fpp"
+    "${CMAKE_CURRENT_LIST_DIR}/OpenSslWrapper.cpp"
+)
+set(MOD_DEPS
+    ${FPRIME_PROJECT_ROOT}/lib/openssl/libcrypto.a
+)
+register_fprime_module()
+target_include_directories(${FPRIME_CURRENT_MODULE} PUBLIC "${FPRIME_PROJECT_ROOT}/lib/openssl/include")
+```
+
+This assumes that the `libcrypto.a` is available for the targeted architecture in the `lib/openssl/` directory of your F´ project root, and that the necessary header files are in `lib/openssl/include/`.
+
+> [!NOTE]
+> If you are attempting to cross-compile but also want to build for your local machine for testing, you may include multiple `.a` files in `lib/openssl/` (one for each target architecture) and use CMake logic to set the `MOD_DEPS` to the correct library file based on the target platform.
```

Comment:
I would just say "you must ensure to have a version of the precompiled library that matches the platform"